main.go

```diff
@@ -13,6 +15,9 @@ import (
 	"github.com/urfave/cli/v2/altsrc"
 )
 
+//go:embed packages/nitro-gui/dist/*
+var staticSiteRaw embed.FS
```

Comment:
I wonder if we can use build tags to optionally enable this. It looks like there is support for conditionally including files based on the build tag: https://www.digitalocean.com/community/tutorials/customizing-go-binaries-with-build-tags

Reply:
Nice find. Let me play around with that.

Reply:
I think it'll work. POC here https://github.com/statechannels/go-nitro/pull/1452/commits/0548dfd8646f6b5ca8c447c3e38d0ca1dcc510ad#diff-2cc94b7e0195048022d70d3f2deac587c20786b5d1effb31e3947a76c233ee2d